features/step_definitions/seeding_steps.rb

```diff
@@ -8,3 +8,8 @@
   SHFProject::Application.load_tasks
   SHFProject::Application.load_seed
 end
+
+Then(/^(\d+) "([^"]*)" records should be created$/) do |number_of_records, models|
+  model_klass = ActiveSupport::Inflector.singularize(models)
+  expect((eval "#{model_klass}").all.size).to eq(number_of_records.to_i)
```

Comment:
yes.  this is an acceptable risk.  (Since it's only executed in a testing environment, and isn't based on user input).  _glad that you're checking to be sure!_